backend/README.md

```diff
@@ -1 +1,8 @@
 # Team Home
```

Comment:
Missing instructions on how to set up a .env file and what environment variables to set in it, along with where to acquire necessary API keys.